coda/coda_mdstore/urls.py

```diff
@@ -9,6 +9,10 @@
     re_path(r'^APP/bag/(?P<identifier>.+?)/$', views.app_bag, name='app-bag-detail'),
     re_path(r'^bag/(?P<identifier>.+?)/$', views.bagHTML, name='bag-detail'),
     re_path(r'^bag/(?P<identifier>ark:\/\d+\/.+?).urls$', views.bagURLList, name='bag-urls'),
+    re_path(
+        r'^bag/(?P<identifier>ark:\/\d+\/.+?)/bagfiles$', views.bagURLList,
+        kwargs={'bagfiles': 'bagfiles'}, name='bag-files'
+    ),
```

Comment:
What do you all think if we change the path of this to `/downloads` or `/links`? Also, since his does not have a file extension, should it have a slash on the end?

Reply:
I'm on board with either of those and also the trailing slack